app/assets/javascripts/initializers/initializeRuntimeBanner.js

```diff
@@ -0,0 +1,62 @@
+/* global Runtime */
+
+function handleDismissRuntimeBanner(event) {
+  const runtimeBanner = document.querySelectorAll('.runtime-banner')[0];
+  if (runtimeBanner) {
+    runtimeBanner.remove();
+  }
+}
+
+// A separate function is needed for this because it's impossible to test a
+// custom scheme launch in Cypress. More about this here:
+// https://medium.com/cypress-io-thailand/understand-stub-spy-apis-that-will-make-your-test-better-than-ever-on-cypress-io-797cb9eb205a
+function launchCustomSchemeDeepLink(targetURL) {
+  window.location.href = targetURL;
+}
+// Making it visible for testing (in order to stub it)
+window.launchCustomSchemeDeepLink = launchCustomSchemeDeepLink;
```

Comment:
This is the best I was able to do in order to test it (check out the `stub` in the Cypress test)

Reply:
It's global by default, so adding it to the window shouldn't be necessary.

e.g. here's some function globally accessible from the base.js

![image](https://user-images.githubusercontent.com/833231/115465961-5e8c8f80-a1fd-11eb-911a-fd288cfb20d0.png)

Reply:
Just tested it locally and you're totally right, tests continue to pass without this so it's unnecessary. I'm removing the window assignment here 👍